adobe-dmeservices/user-sync.py
# Sign refactor in client.py

## Summary

* Refactor Exception Handling statement in client.py file, so that all Assertion Exception is in the incorrect format.
changes are in the below methods in client.py

1.  deactivate_user() 
2.  insert_user()
3.  update_user()

## Testing Steps
* Execute sign sync and check output format
* Execute test_is_directory_user_in_groups and check response using code coverage

Fixes #180 
Fixes #182
